hub/templates/hub/area.html

```diff
@@ -1,4 +1,3 @@
-{% extends 'hub/base.html' %}
```

Comment:
I think this line has been removed accidentally? It obviously breaks the page layout.